storage/innobase/xtrabackup/doc/source/prerequisites/permissions.rst

```diff
@@ -0,0 +1,134 @@
+
+For the purposes of this manual, *permissions* refers to the ability of a user to access and perform changes on the relevant parts of the host's filesystem, such as starting or stopping services, or installing software.
+
+*Privileges* refer to the abilities of a database user to perform different kinds of actions on the database server.
+
+Permissions at a system level
+---------------------------------------------
+
+Once connected to the server, in order to perform a backup you must have the
+``READ`` and ``EXECUTE`` permissions at a filesystem level in the
+server's `datadir`.
+
+Checking for Permissions
++++++++++++++++++++++++++++++++
+
+There are many ways for checking the permission on a file or directory. For example, ``ls -ls /path/to/file`` or ``stat /path/to/file | grep Access`` will do the job: ::
+
+  $ stat /etc/mysql | grep Access
+  Access: (0755/drwxr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
+  Access: 2011-05-12 21:19:07.129850437 -0300
+  $ ls -ld /etc/mysql/my.cnf 
+  -rw-r--r-- 1 root root 4703 Apr  5 06:26 /etc/mysql/my.cnf
+
+As in this example, ``my.cnf`` is owned by ``root`` and not writable for anyone else. Assuming that you do not have the password for ``root``, you can check what permissions you have on this file type with ``sudo -l``: ::
+
+  $ sudo -l
+  Password:
+  You may run the following commands on this host:
+  (root) /usr/bin/
+  (root) NOPASSWD: /etc/init.d/mysqld
+  (root) NOPASSWD: /bin/vi /etc/mysql/my.cnf
+  (root) NOPASSWD: /usr/local/bin/top
+  (root) NOPASSWD: /usr/bin/ls
+  (root) /bin/tail
+
+Using ``sudo`` in ``/etc/init.d/``, ``/etc/rc.d/`` or ``/sbin/service`` script provides the ability to start and stop services.
+
+Also, if you can execute the package manager of your distribution, you can install or remove software with it. If not, having ``rwx`` permission over a directory will let you do a local installation of software by compiling it there. This is a typical situation in many companies.
+
+There are other ways for managing permissions, such as using *PolicyKit*, *Extended ACLs* or *SELinux*, which may be preventing or allowing your access. You should check them in that case.
+
+Privileges Needed
+------------------------------
+
+The database user needs the following privileges on the tables or databases to be backed up:
+
+* ``RELOAD`` and ``LOCK TABLES`` (unless the `--no-lock <--no-lock>`
+  option is specified) in order to run :mysql:`FLUSH TABLES WITH READ LOCK` and
+  :mysql:`FLUSH ENGINE LOGS` prior to start copying the files, and requires this
+  privilege when `Backup Locks
+  <http://www.percona.com/doc/percona-server/8.0/management/backup_locks.html>`_
+  are used
+
+* ``BACKUP_ADMIN`` privilege is needed to query the
+  performance_schema.log_status table, and run :mysql:`LOCK INSTANCE FOR BACKUP`,
+  :mysql:`LOCK BINLOG FOR BACKUP`, or :mysql:`LOCK TABLES FOR BACKUP`.
+
+* ``REPLICATION CLIENT`` in order to obtain the binary log position,
```

Comment:
``REPLICATION CLIENT``privilege — to obtain

Reply:
Updated.